test_scripts/Smoke/HeartBeat/ATF_HeartBeat_App_does_not_send_HB_and_does_not_respond.lua

```diff
@@ -0,0 +1,119 @@
+require('cardinalities')
+require('user_modules/AppTypes')
+
+-- [[Local variables]]
+local default_app_params = config.application1.registerAppInterfaceParams
+local default_app_params2 = config.application2.registerAppInterfaceParams
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+commonPreconditions:BackupFile("smartDeviceLink.ini")
+
+function Test:StartSDL_And_Connect_Mobile()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      self:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        self:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+        end)
+      end)
+    end)
+  end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:Start_Session_And_Register_App()
+  self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+  commonFunctions:write_parameter_to_smart_device_link_ini("HeartBeatTimeout", 5000)
+  self.mobileSession.sendHeartbeatToSDL = false
+  self.mobileSession.answerHeartbeatFromSDL = false
+  self.mobileSession.ignoreSDLHeartBeatACK = false
+  self.mobileSession:StartRPC():Do(function()
+    local correlation_id = self.mobileSession:SendRPC("RegisterAppInterface", default_app_params)
+    EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", 
+      { application = { appName = default_app_params.appName}}):Do(function(_,data)
+      default_app_params.hmi_app_id = data.params.application.appID
+    end)
+    self.mobileSession:ExpectResponse(correlation_id, {success = true, resultCode = "SUCCESS"})
+    self.mobileSession:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN"})
+    self.mobileSession:ExpectNotification("OnPermissionsChange", {})  
+  end)
+end
+
+function Test:Register_Second_App_With_HeartBeat()
+  self.mobileSession1 = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession1.sendHeartbeatToSDL = true
+  self.mobileSession1.answerHeartbeatFromSDL = true
+  self.mobileSession1.ignoreSDLHeartBeatACK = false
+  self.mobileSession1:StartRPC():Do(function()
+    local correlation_id = self.mobileSession1:SendRPC("RegisterAppInterface", default_app_params2)
+    EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = default_app_params2.appName}})
+    self.mobileSession1:ExpectResponse(correlation_id, {success = true, resultCode = "SUCCESS"})
+    self.mobileSession1:ExpectNotification("OnHMIStatus", {hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN"})
+    self.mobileSession1:ExpectNotification("OnPermissionsChange", {})  
+  end)
+end
+
+function Test:Wait_15_seconds_And_Verify_OnAppUnregistered()
```

Comment:
@istoimenova but the main idea of the script it is that SDL must close session and application will be unregister, so if we received notification OnAppUnregistered, there is no point in waiting more